pkg/cli/gen.go

```diff
@@ -239,7 +240,18 @@ Output the list of cluster settings known to this binary.
 					defaultVal = override
 				}
 			}
-			row := []string{wrapCode(name), typ, wrapCode(defaultVal), setting.Description()}
+
+			settingDesc := setting.Description()
+			if strings.Contains(name, "sql.defaults") {
+				settingDesc = fmt.Sprintf(`%s
+This cluster setting is being kept to preserve backwards-compatibility.
+This session variable default can now be configured using ALTER ROLE: %s`,
```

Comment:
Maybe `This session variable default should now be configured using ALTER ROLE ... SET: `